ci/test-pytest.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+cd "$( dirname "${BASH_SOURCE[0]}" )"/..
+
+sudo -H pip3 install pipenv
```

Comment:
Why the `-H`? Also, in dockerfiles, we do `pipenv install --system --deploy --ignore-pipfile`, can we do the same here?